test/unit/spec/EnrollWindowsHello_spec.js

```diff
@@ -126,24 +141,11 @@ function (Okta,
         return emulateWindows()
         .then(setup)
         .then(function (test) {
-          expect(test.form.hasErrorNotWindows()).toBe(false);
+          expect(test.form.hasErrorWindowsHello()).toBe(false);
           expect(test.form.submitButton().length).toBe(1);
         });
       });
 
-      itp('subtitle changes after submitting the form', function () {
```

Comment:
We are removing this, but we are not replacing it with tests to cover the changed functionality - anything to do with our actual usage of the spinner, or the various views that show up with different enrollment states. This seems like the main purpose of the PR - we should add test coverage for new functionality that we're adding.